testsuite/unittests/test_rdv_param_setting.sli

```diff
@@ -244,20 +244,39 @@ statusdict /have_gsl :: {
   /MT19937 /uniform_int << /low 11 /high 10 >> run_test
 } fail_or_die
 
+/:range_limit
```

Comment:
I am not sure I follow what this function does. 
Can you maybe comment the parameter?